packages/itwinui-css/src/progress-indicator/radial.scss

```diff
@@ -4,148 +4,109 @@
 @use '../mixins';
 
 @mixin iui-progress-indicator-radial {
-  position: relative;
-  display: inline-block;
   --_iui-progress-indicator-radial-size: #{map.get(mixins.$iui-icon-sizes, xl)};
-
+  position: relative;
+  vertical-align: middle;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
   width: var(--_iui-progress-indicator-radial-size);
   height: var(--_iui-progress-indicator-radial-size);
+  border-radius: 50%;
```

Comment:
Why not using border radius variable?

Reply:
i think it was already like this before. but changed